scripts/macos_3_prepare_before_sign.sh

```diff
@@ -8,5 +8,5 @@ cd ..
 rm -rf temp_macos
 mkdir temp_macos
 
-cp -R build/macos/Build/Products/Release/Rune.app temp_macos
```

Comment:
It's still being copied the next line;

This specific change is to use `ditto` to copy `.app` bundles instead of `cp -R`, as `ditto` correctly preserves symbol links and macOS APFS permissions and ACL. This practice is recommended by Apple.